frontend/README.md

```diff
@@ -1,27 +1,117 @@
-# React + TypeScript + Vite
+# Frontend README
 
-This template provides a minimal setup to get React working in Vite with HMR and some ESLint rules.
+[Introduction](#introduction) •
+[Getting Started](#getting-started) •
+[Tests](#tests) •
+[Tech Stack](#tech-stack)
 
-Currently, two official plugins are available:
+# Introduction <a name="introduction"/>
 
-- [@vitejs/plugin-react](https://github.com/vitejs/vite-plugin-react/blob/main/packages/plugin-react/README.md) uses [Babel](https://babeljs.io/) for Fast Refresh
-- [@vitejs/plugin-react-swc](https://github.com/vitejs/vite-plugin-react-swc) uses [SWC](https://swc.rs/) for Fast Refresh
+This document provides an overview of the frontend part of our application. It outlines how to get started, guides on tests and the technologies used.
 
-## Expanding the ESLint configuration
+## Getting started <a name="getting-started"/>
 
-If you are developing a production application, we recommend updating the configuration to enable type aware lint rules:
+Before you begin, ensure you have the following installed:
 
-- Configure the top-level `parserOptions` property like this:
+- Node.js (version 18 or later)
+- npm (for managing packages)
```

Comment:
Because `npm run dev` will only generate the frontend side without any backend data, we'll need instruction to set up the backend. I will also add such instructions to README in the root directory, so please add a third bullet point:
```
- Start the customized `zeebe-test-bench` and build the application as instructed in the main README.
```